interacting-dom/app.js

```diff
@@ -0,0 +1,58 @@
+const btnCreate = document.getElementById("btn-main");
+const btnToggle = document.querySelector(".btn-toggle");
+const btnRemove = document.querySelector(".btn-remove");
+const taskList = document.querySelector(".list-container ul");
+const remove = document.querySelectorAll('li');
+for (let i = 0; i < remove.length; i++) {
+    attachRemoveButton(remove[i]);
+}
+
+function attachRemoveButton(poop) {
+    let remove = document.createElement('button');
+    remove.className = 'remove';
+    remove.textContent = 'Remove';
+    poop.appendChild(remove);
+}
+
+// const listItem = document.getElementsByTagName("li");
+
+// // for (let i = 0; i < listItem.length; i += 1) {
```

Comment:
Scan the code for commented out code and remove those before doing a PR.

Reply:
Done